### DISCUSSION TITLE
Where are offline songs stored?

### DISCUSSION THREAD

**Discussion by antoni0C0des** [gokadzev/Musify#356] [Q&A] [answered]
Offline song downloads are super neat but the app really doesn't have a way to arrange them. I thought of using a third party music player that would fetch files from musify but I have no idea where are they stored. Is this even possible?

**Comment by gokadzev** [comment, marked as answer]
> Offline song downloads are super neat but the app really doesn't have a way to arrange them. I thought of using a third party music player that would fetch files from musify but I have no idea where are they stored. Is this even possible?

No, offline music in Musify can only be played within Musify. We implemented a download feature that initially saved files accessible to other apps. However, after several discussions and encountering issues, we concluded that it would be better this way.

**Reply by schklom** [reply]
From what you wrote, I think the previous download feature was replaced by the new "Make offline available" button feature?
If so, would you be open to add an (maybe opt-in) Export button available for recent Android versions, instead of a replacement to the current "Make offline available" button? I mean both could co-exist.
That way, the "Make offline available" would work as currently, and the Export button would be an extra button that can be enabled and should work for most users but may be buggy.
